massa-versioning/src/lib.rs

```diff
@@ -45,6 +45,8 @@
 //!
 //! Unit tests in versioning_factory.rs shows a basic but realistic implementation of a AddressFactory (impl the Factory trait)
 
+pub mod address_factory;
```

Comment:
Maybe we can separate the factory implementations in a folder?

Reply:
I personally prefer good file naming over sub directories in a crate, if the team shares your opinion I don't mind changing it.